Lib/test/test_pickletools.py

```diff
@@ -40,13 +40,9 @@ def test_in_band_buffers(self): # TODO: RUSTPYTHON, remove when this passes
     def test_nested_names(self): # TODO: RUSTPYTHON, remove when this passes
         super().test_nested_names()
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
When you delete this, can you also delete the two lines that say `# TODO: RUSTPYTHON, remove when this passes`?